tests/cancun/eip4844_blobs/test_blob_txs_full.py

```diff
@@ -237,17 +241,12 @@ def generate_full_blob_tests(
     Return a list of tests for invalid blob transactions due to insufficient max fee per blob gas
     parametrized for each different fork.
     """
-    blob_size = Spec.FIELD_ELEMENTS_PER_BLOB * SpecHelpers.BYTES_PER_FIELD_ELEMENT
     max_blobs = fork.max_blobs_per_block()
     return [
         pytest.param(
             [  # Txs
                 [  # Blobs per transaction
-                    Blob(
-                        data=bytes(blob_size),
-                        kzg_commitment=INF_POINT,
-                        kzg_proof=INF_POINT,
-                    ),
+                    Blob.from_fork(Cancun),
```

Comment:
```suggestion
                    Blob.from_fork(fork),
```
We can just pass the fork here to avoid constraining the test to a hard-coded fork.